packages/text-area/src/TextArea.story.tsx

```diff
@@ -20,22 +31,28 @@ export default {
 };
 
 const Template: ComponentStory<typeof StoryTextArea> = ({
+  baseFontSize,
   darkMode,
+  lgProviderBaseFontSize,
   ...args
-}: TextAreaProps) => (
-  <div
-    className={css`
-      padding: 30px;
-      background-color: ${darkMode ? palette.black : 'white'};
-    `}
-  >
-    <TextArea darkMode={darkMode} {...args} />
-  </div>
+}: TextAreaProps & { lgProviderBaseFontSize: 14 | 16 }) => (
```

Comment:
Try removing this type def since it might already be picked up from `ComponentStory<typeof StoryTextArea>`

Reply:
I get a prop-type error if i remove this but I do notice that you use `eslint-disable-next-line react/prop-types` to get rid of this same error in other storybook files.